moveit_planners/ompl/ompl_interface/CMakeLists.txt

```
@@ -1,5 +1,10 @@
 set(MOVEIT_LIB_NAME moveit_ompl_interface)
 
+if (APPLE)
```

To fix `ament_lint_cmake` error
```suggestion
if(APPLE)
```